tests/test_mdn.py

```diff
@@ -120,53 +120,69 @@ def test_classification(
             )
             tabular_model.fit(train=train, test=test)
 
-# import numpy as np
-# import pandas as pd
-# from sklearn.datasets import fetch_california_housing, fetch_covtype
-
- 
-# def regression_data():
-#     dataset = fetch_california_housing(data_home="data", as_frame=True)
-#     df = dataset.frame.sample(5000)
-#     df["HouseAgeBin"] = pd.qcut(df["HouseAge"], q=4)
-#     df["HouseAgeBin"] = "age_" + df.HouseAgeBin.cat.codes.astype(str)
-#     test_idx = df.sample(int(0.2 * len(df)), random_state=42).index
-#     test = df[df.index.isin(test_idx)]
-#     train = df[~df.index.isin(test_idx)]
-#     return (train, test, dataset.target_names)
-
-
-# def classification_data():
-#     dataset = fetch_covtype(data_home="data")
-#     data = np.hstack([dataset.data, dataset.target.reshape(-1, 1)])[:10000, :]
-#     col_names = [f"feature_{i}" for i in range(data.shape[-1])]
-#     col_names[-1] = "target"
-#     data = pd.DataFrame(data, columns=col_names)
-#     data["feature_0_cat"] = pd.qcut(data["feature_0"], q=4)
-#     data["feature_0_cat"] = "feature_0_" + data.feature_0_cat.cat.codes.astype(str)
-#     test_idx = data.sample(int(0.2 * len(data)), random_state=42).index
-#     test = data[data.index.isin(test_idx)]
-#     train = data[~data.index.isin(test_idx)]
-#     return (train, test, ["target"])
 
+@pytest.mark.parametrize("multi_target", [False])
+@pytest.mark.parametrize(
+    "continuous_cols",
+    [
+        [
+            "AveRooms",
+            "AveBedrms",
+            "Population",
+            "AveOccup",
+            "Latitude",
+            "Longitude",
+        ]
+    ],
+)
+@pytest.mark.parametrize("categorical_cols", [["HouseAgeBin"]])
+@pytest.mark.parametrize("continuous_feature_transform", [None])
+@pytest.mark.parametrize("normalize_continuous_features", [True])
+@pytest.mark.parametrize("variant", [CategoryEmbeddingMDNConfig, NODEMDNConfig, AutoIntMDNConfig])
+@pytest.mark.parametrize("num_gaussian", [1, 2])
+@pytest.mark.parametrize("ssl_task", ["Denoising", "Contrastive"])
+@pytest.mark.parametrize("aug_task", ["cutmix", "mixup"])
+def test_ssl(
```

Comment:
No need for test_ssl for MDN

Reply:
I have keep only one test in order to check the raise of the exception in case of ssl task